releases/v1.5.0.toml

```
@@ -147,4 +147,14 @@ To check version, see the version value in the containerd toml configuration.
 version=2
 ```
 
+#### FreeBSD Runtime Support (Experimental)
+
+This release includes changes that allow containerd to run on FreeBSD with a
+compatible runtime, such as [runj](https://github.com/samuelkarp/runj). This
+support should be considered experimental and currently will not be included in
+binaries releases. The runtimes used by containerd are maintained separately and
```

"binaries releases" seems a bit awkward to me.

```suggestion
binary releases. The runtimes used by containerd are maintained separately and
```

---

This support should be considered experimental. The `runj` binary is not included in this release.

---

I had a lot of trouble getting the wording right here, its more that we won't be officially releasing any builds for freebsd at this time.